pkg/distro/generic/imagetype.go

```diff
@@ -160,7 +159,9 @@ func (t *imageType) getPartitionTable(customizations *blueprint.Customizations,
 	}
 
 	partitioningMode := options.PartitioningMode
-	if t.ImageTypeYAML.RPMOSTree {
+	// XXX: this needs to go into YAML too :(
+	// XXX2: this was also not covered by the manifests tests, why?
+	if t.arch.distro.DistroYAML.DistroLike == manifest.DISTRO_FEDORA && t.ImageTypeYAML.RPMOSTree {
```

Comment:
This should probably just become an `unsupported_partitioning_mode` now, once https://github.com/osbuild/images/pull/1653 is merged at least we have basic testing for this.